### DISCUSSION TITLE
No matching distribution found for sktime~=0.11.4

### DISCUSSION THREAD

**Discussion by AadityaH** [pycaret/pycaret#3020] [Q&A]
Hello All,  

I am receiving following error while I am trying to deploy app based on streamlit in heroku. 

ERROR: No matching distribution found for sktime~=0.11.4 

I have referred various forums but could not find a solution. 

PyCaret version I have is pycaret==3.0.0rc3

your help in this error is highly appreciated 

Please let me know if any additional information is required. 

@moezali1 @Yard1 @ngupta23

**Comment by ngupta23** [comment]
pycaret==3.0.0rc3 requires sktime 0.13.2. Please make sure that version is installed.

**Reply by AadityaH** [reply]
Thank you Nikhil for your quick reply :)

Is there a way by which I can get list of libraries [ versions] that are compatible with PyCaret==3.0.0rc3 , I will install those versions in specific environment.

I am primarily using pandas , numpy & streamlit

Thanks & appreciate your feedback.

**Reply by AadityaH** [reply]
> pycaret==3.0.0rc3 requires sktime 0.13.2. Please make sure that version is installed.

Hi Nikhil @ngupta23 ,

I tried changing the sktime version to 0.13.2 , however Error still persists - details as follows :

**The user requested sktime==0.13.2
           pycaret 3.0.0rc3 depends on sktime~=0.11.4**

if I remove sktime from requirements.txt file , I get following error : 

ERROR: No matching distribution found for sktime==0.11.4
 !     Push rejected, failed to compile Python app.
 !     Push failed

can you please have a look? Repo link as follows if needed : https://github.com/AadityaH/demandforecasting

**Reply by ngupta23** [reply]
What happens when you just keep the following in your requirements file and remove the rest (I changed it to rc4)?

pycaret==3.0.0rc4
streamlit==1.11.1
streamlit_lottie==0.0.3

**Reply by AadityaH** [reply]
It worked - Deployed successfully :) 

Thanks. Appreciate your help ! :)
